instance/models/instance.py

```diff
@@ -234,6 +235,15 @@ def github_base_url(self):
         return 'https://github.com/{0.fork_name}'.format(self)
 
     @property
+    def github_pr_url(self):
+        """
+        Web URL of the Github PR, or None if the PR number is not set.
+        """
+        if self.github_pr_number is None:
+            return None
+        return '{0.github_base_url}/pull/{0.github_pr_number}'.format(self)
```

Comment:
@smarnach I just realized, testing on the real set of sandboxes, that there is a bug here - `github_base_url` is the base URL of the repo being deployed - not the repo the PR is opened on. This works well when they are the same, but not when the PR is opened from a fork. 

This should be using `settings.WATCH_FORK` or - likely better since we'll eventually watch mutiple forks - store the full PR URL instead of just the number. Since this is already merged and the ticket closed, I'll add a ticket in the backlog about this.